modules/renter/hostdb/persist.go

```diff
@@ -17,8 +17,8 @@ func (hdb *HostDB) persistData() hdbPersist {
 	for _, entry := range hdb.allHosts {
 		data.AllHosts = append(data.AllHosts, *entry)
 	}
-	for _, node := range hdb.activeHosts {
-		data.ActiveHosts = append(data.ActiveHosts, *node.hostEntry)
+	for _, entry := range hdb.activeHosts {
+		data.ActiveHosts = append(data.ActiveHosts, *entry)
```

Comment:
just want to confirm that this saves everything important.

Can you verify that there is a test for saving and loading hosts, including things like the scan history?